portfolio/src/main/webapp/script.js

```diff
@@ -20,11 +20,21 @@ function getRandomFact() {
       ['I am minoring in Drama.', 'My favorite color is teal.', 'I love crime shows.', 'I love superhero movies.',
       'I am on the bhangra team at Carnegie Mellon University.'];
 
-
   // Pick a random greeting.
   const fact = facts[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+//dealing with image gallery
```

Comment:
nit: Add a space between double slash and dealing.

Reply:
Okay, will fix those throughout.